src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -1,85 +1,194 @@
+        ExtractableResponse<Response> 화곡역 = 지하철_노선_생성_요청(화곡);
+        LineRequest 까치산 = 지하철_노선_정보("까치산", "purple");
 
         // when
-        // 지하철_노선_수정_요청
+        ExtractableResponse<Response> 수정 = 지하철_노선_수정_요청(화곡역, 까치산);
 
         // then
-        // 지하철_노선_수정됨
+        지하철_노선_응답됨(수정);
     }
 
     @DisplayName("지하철 노선을 제거한다.")
     @Test
     void deleteLine() {
         // given
         // 지하철_노선_등록되어_있음
+        LineRequest 화곡 = 지하철_노선_정보("화곡", "purple");
+        ExtractableResponse<Response> 화곡역 = 지하철_노선_생성_요청(화곡);
 
         // when
-        // 지하철_노선_제거_요청
+        ExtractableResponse<Response> 삭제 = 지하철_노선_제거_요청(화곡역);
 
         // then
-        // 지하철_노선_삭제됨
+        지하철_노선_삭제됨(삭제);
+    }
+
+    private LineRequest 지하철_노선_정보(String name, String color) {
+        return new LineRequest(name, color);
+    }
+
+    private ExtractableResponse<Response> 지하철_노선_생성_요청(LineRequest lineRequest) {
+        return RestAssured
+            .given().log().all()
+            .body(lineRequest)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when().post(LINES_PATH)
+            .then().log().all().extract();
+    }
+
+    private ExtractableResponse<Response> 지하철_노선_목록_조회() {
+        return RestAssured
+            .given().log().all()
+            .when().get(LINES_PATH)
+            .then().log().all().extract();
+    }
+
+    private ExtractableResponse<Response> 지하철_노선_조회_요청(ExtractableResponse<Response> response) {
+        return RestAssured
+            .given().log().all()
+            .body(response.header("Location"))
+            .when().get(LINES_PATH)
+            .then().log().all().extract();
+    }
+
+    private ExtractableResponse<Response> 지하철_노선_수정_요청(ExtractableResponse<Response> response,
+        LineRequest changeLine) {
+        return RestAssured
+            .given().log().all()
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .body(changeLine)
+            .when().put(response.header("Location"))
+            .then().log().all().extract();
+    }
+
+    private ExtractableResponse<Response> 지하철_노선_제거_요청(ExtractableResponse<Response> response) {
+        return RestAssured
+            .given().log().all()
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when().delete(response.header("Location"))
+            .then().log().all().extract();
+    }
+
+    private void 지하철_노선_목록_포함됨(ExtractableResponse<Response> resultResponse,
+        ExtractableResponse<Response> response1, ExtractableResponse<Response> response2) {
+
+        List<Long> expectedLineIds = Arrays.asList(response1, response2).stream()
+            .map(it -> Long.parseLong(it.header("Location").split("/")[2]))
+            .collect(Collectors.toList());
+
+        List<Long> resultLineIds = resultResponse.jsonPath().getList(".", LineResponse.class).stream()
+            .map(it -> it.getId())
+            .collect(Collectors.toList());
```

Comment:
이 부분은 아래와 같이 간단하게 줄여볼 수 있습니다.
```suggestion
        List<Long> resultLineIds = resultResponse.jsonPath().getList("id", Long.class)
```
`restassured`의 기능들을 잘 활용하면 더욱 간결한 인수테스트를 작성할 수 있습니다! 😊

Reply:
오오 .. 감사합니다 한번 찾아봐야겠네요 !